src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
```

Comment:
Reading through the code of this file, I got confused by at least three different names used in this file for a very similar concept:

1) CrossReference
2) ExternalLinks
3) Xrefs

I wonder whether using two variables clearly showing their relationships to each other, e.g. `crossReference` and `crossReferenceGroup`, where crossReferenceGroup would have the type of:

```
{
  source: Source;
  references: CrossReferenceWithoutSource
}
```
would not make the code clearer.

Reply:
Thanks. That's done now.